src/Field.php

```diff
@@ -21,6 +21,11 @@ function __construct($defaults = []) {
         }
     }
 
+    public function get()
+    {
+        return $this->owner[$this->short_name];
```

Comment:
because we need the value of the field, not the actual field itself. This is same as 4.3 btw.